shumai/io/network.ts

```diff
@@ -244,6 +252,7 @@ export type RouteStats = {
 export function serve(request_dict: Record<string, any>, options: ServeOpts) {
   const user_data = {}
   const statistics: Record<string, RouteStats> = {}
+  const op_stats: TensorOpStats = {}
```

Comment:
I suspect we can get rid of this entirely if serve_request returns the statistics

Reply:
I can submit this change as a separate PR this evening!